docs/cmd-generate-contributing.md

```diff
@@ -0,0 +1,78 @@
+# dephell generate contributing
+
+Adds `CONTRIBUTING.md` for your project.
+
+Output example:
```

Comment:
I think, it's better to drop that example from here. We will improve it a much, and it's difficult to keep it always actual. What do you think?